src/Db/Diff/PKSimDB_diff.txt

```diff
@@ -1 +1,42 @@
+INSERT INTO tab_container_names(container_type,container_name,display_name,description,extension,sequence,icon_name) VALUES('DISEASE_STATE','CKD','CKD Parameter container','CKD Parameter container',NULL,1,NULL);
+INSERT INTO tab_container_parameter_values(parameter_value_version,species,container_id,container_type,container_name,parameter_name,default_value,min_value,min_isallowed,max_value,max_isallowed,value_origin) VALUES('DiseaseStates','Human',5954,'DISEASE_STATE','CKD','InputGFR',60.0,0.0,1,60.0,1,92);
+INSERT INTO tab_container_parameters(container_id,container_type,container_name,parameter_name,can_be_varied,read_only,visible,can_be_varied_in_population,group_name,sequence,build_mode,building_block_type,is_input) VALUES(5954,'DISEASE_STATE','CKD','InputGFR',0,0,1,0,'BODY_ANATOMY',1,'LOCAL','INDIVIDUAL',1);
+INSERT INTO tab_container_types(container_type,display_name,description) VALUES('DISEASE_STATE','Disease State','Disease state container');
+INSERT INTO tab_containers(container_id,container_type,container_name,parent_container_id,parent_container_type,parent_container_name,visible,is_logical) VALUES(5954,'DISEASE_STATE','CKD',NULL,NULL,NULL,1,1);
+INSERT INTO tab_dimensions(dimension,kernel_unit) VALUES('Flow per surface area','l/min/1.73m²');
+CREATE TABLE tab_disease_states
+(
+    disease_state  text not null
+        constraint tab_disease_states_pk
+            primary key,
+    display_name   text not null,
+    description    text not null,
+    implementation text
+);
+INSERT INTO tab_disease_states(disease_state,display_name,description,implementation) VALUES('CKD','CKD Display TODO','CKD Description TODO','CKDDiseaseState');
+CREATE UNIQUE INDEX tab_disease_states_description_uindex
+    on tab_disease_states (description);
+CREATE UNIQUE INDEX tab_disease_states_disease_state_uindex
+    on tab_disease_states (disease_state);
+CREATE UNIQUE INDEX tab_disease_states_display_name_uindex
+    on tab_disease_states (display_name);
+INSERT INTO tab_parameter_value_versions(parameter_value_version,display_name,description,category) VALUES('DiseaseStates','Diseases States','Disease states parameters','DiseaseStates');
+INSERT INTO tab_parameters(parameter_name,display_name,description,dimension,default_unit) VALUES('InputGFR','InputGFR TODO','InputGFR TODO','Flow per surface area',NULL);
+CREATE TABLE tab_population_disease_states
+(
+    population    text not null
+        constraint tab_population_disease_states_tab_populations_population_fk
+            references tab_populations
+            on update cascade,
+    disease_state text not null
+        constraint tab_population_disease_states_tab_disease_states_disease_state_fk
+            references tab_disease_states
+            on update cascade,
+    constraint tab_population_disease_states_pk
+        primary key (population, disease_state)
+);
+INSERT INTO tab_population_disease_states(population,disease_state) VALUES('European_ICRP_2002','CKD');

```

Comment:
only ICRP? no other population?

Reply:
No all of them. I just wanted to add one to make sure the strucutre was ok. All of them now